tests/controllers/buyerReview.test.ts

```diff
@@ -0,0 +1,141 @@
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import mongoose from 'mongoose';
+
+import BuyerReview from '../../src/models/buyerReview';
+
+import {
+  mockBuyerUser,
+  mockSellerUser,
+  mockSellerService,
+  mockBuyerRequest,
+  mockBuyerReview,
+} from '../helpers/buyerReview';
+
+import {
+  getAllBuyerReviews,
+  getBuyerReviewById,
+  getBuyerReviewsByBuyerUser,
+  getBuyerReviewsByPostingUser,
+  createBuyerReview,
+  deleteBuyerReview,
+} from '../../src/controllers/buyerReview';
+
+describe('Controller: BuyerReview', () => {
+  let mongoServer: MongoMemoryServer;
+
+  beforeAll(async () => {
+    mongoServer = await MongoMemoryServer.create();
+
+    await mongoose.connect(mongoServer.getUri());
+
+    await mockBuyerUser.save();
+    await mockSellerUser.save();
+    await mockSellerService.save();
+    await mockBuyerRequest.save();
+    await mockBuyerReview.save();
+  });
+
+  afterAll(async () => {
+    if (mongoServer) {
+      await mongoose.connection.dropDatabase();
+      await mongoose.connection.close();
+      await mongoServer.stop();
+    }
+  });
+
+  it('Should be able to fetch all BuyerReviews', async () => {
+    let getAllBuyerReviewsQuery = await getAllBuyerReviews();
+    expect(getAllBuyerReviewsQuery.status).toBe(true);
+    expect(getAllBuyerReviewsQuery.data).toBeInstanceOf(Array);
+    expect(getAllBuyerReviewsQuery.data?.length).toBe(1);
+    expect(getAllBuyerReviewsQuery.data?.at(0)?._id).toEqual(
+      mockBuyerReview._id
+    );
```

Comment:
I would recommend inserting another mock and then also checking if 2 BuyerReviews have been returned.

Reply:
Done